cmd/config.go

```diff
@@ -139,6 +151,12 @@ func config(ctx *cli.Context) error {
 				return fmt.Errorf("Aborted.")
 			}
 		}
+		if clientVer && format.CheckVersion() != nil {
+			warn("Clients with the same version of this will be rejected after modification.")
```

Comment:
Should we check the version of all active sessions?

Reply:
IMO that's too much for `config` command